### DISCUSSION TITLE
事件，也就是脚本，的讲解

### DISCUSSION THREAD

**Discussion by guotong1988** [palxex/palresearch#11] [General]
<img width="950" alt="WX20200121-134843@2x" src="https://user-images.githubusercontent.com/4702353/72778846-09a64600-3c55-11ea-8d92-eb8859ecd7cf.png">

**Comment by guotong1988** [comment]
![image](https://user-images.githubusercontent.com/4702353/73038901-10200200-3e8f-11ea-83f6-6b9f84d05425.png)

**Comment by zxyyangxz** [comment]
一个小白，可以讲解下98版扬州太守赏金的问题吗
